File: RateManagerCss/Models/OtaRequest/BaseGuestAmount.vb
Public Class BaseGuestAmount

    Private amtBeforeTax As String
    Public Property AmountBeforeTax() As String
        Get
            Return amtBeforeTax
        End Get
        Set(ByVal value As String)
            amtBeforeTax = value
        End Set
    End Property

End Class

File: APIServices/Conflux/Models/OTA/Rates/BaseGuestAmount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace APIServices.Conflux.OTA.Models.Rates
{
    public class BaseGuestAmount
    {
        public decimal? AmountBeforeTax { get; set; }
        public decimal? AmountAfterTax { get; set; }
        public string NumberOfGuests { get; set; }
        public int? AgeQualifyingCode { get; set; }
        //public string CurrencyCode { get; set; }
    }
}
